### PR TITLE
Sets `DEFAULT_AUTO_FIELD` for autogenerated PKs

### PR DESCRIPTION
**Pull request by kingdomlevel**
- Sets default strategy for auto-generated PKs to be `BigAutoField` (`bigint` in the DB)
- Creates migration to update all models in the `assessment` app to use this for their `id` field

### CODE REVIEW

**Review by joel-stobart — APPROVED** [review]
That looks good

**Comment by kingdomlevel** [comment]
Ideally, we should wait until #84 and #83 are merged before merging this to develop...

**Pull request merged**
